src/ode_default_alg.jl

```diff
@@ -31,15 +31,25 @@ function default_algorithm{uType,tType,inplace}(prob::AbstractODEProblem{uType,t
       end
     end
   else # The problem is stiff
-    if uEltype <: Float64 # Sundials only works on Float64!
-      if tol_level == :high_tol || callbacks || mm # But not in these cases
-        alg = Rosenbrock23()
-      else
-        alg = Rodas4()
-      end
+    if uType <: Array{Float64} && !mm && length(prob.u0) > 1000
+      # Sundials only works on Float64!
+      # Sundials is fast when problems are large enough
+      alg = CVODE_BDF()
+    elseif uType <: Array{Float64} && !mm && length(prob.u0) > 10000
```

Comment:
You can't get here

Reply:
oh true, I need to flip those. Thanks.

Reply:
no problems, always good with an extra pair of eyes!

Reply:
yup! This is something that should get more tests as well, though I think that it's still somewhat in flux.